apps/api/v2/src/ee/bookings/2024-08-13/controllers/bookings.controller.ts

```diff
@@ -542,4 +543,19 @@ export class BookingsController_2024_08_13 {
       data: bookingReferences,
     };
   }
+
+  @Get("/:bookingUid/sessions")
```

Comment:
The endpoint name could be improved to remove a bit of vagueness

How about /:bookingUid/conferencing-sessions